functest/new_api_test.go

```diff
@@ -94,30 +98,31 @@ func customSend(data string) (map[string]interface{}, error) {
 func TestEmptyBody(t *testing.T) {
 	res, err := customSend("")
 	require.NoError(t, err)
-	require.Equal(t, "[ UnmarshalRequest ] Empty body", res["error"])
+	require.Equal(t, "[ UnmarshalRequest ] Empty body", res["error"].(map[string]interface{})["message"].(string))
 }
 
 func TestCrazyJSON(t *testing.T) {
 	res, err := customSend("[dh")
 	require.NoError(t, err)
-	require.Contains(t, res["error"], "[ UnmarshalRequest ] Can't unmarshal input params: invalid")
+	require.Contains(t, res["error"].(map[string]interface{})["message"].(string), "[ UnmarshalRequest ] Can't unmarshal input params: invalid")
 }
 
 func TestIncorrectSign(t *testing.T) {
-	args, err := insolar.MarshalArgs(nil)
-	require.NoError(t, err)
 	seed, err := requester.GetSeed(TestAPIURL)
 	require.NoError(t, err)
-	body, err := requester.GetResponseBody(TestCallUrl, requester.PostParams{
-		"params":    args,
-		"method":    "SomeMethod",
-		"reference": root.ref,
-		"seed":      seed,
-		"signature": []byte("1234567890"),
-	})
+	body, err := requester.GetResponseBodyContract(
+		TestCallUrl,
+		requester.Request{
+			JSONRPC: "2.0",
+			ID:      1,
+			Method:  "SomeMethod",
+			Params:  requester.Params{Seed: seed},
+		},
+		"1234567890",
+	)
 	require.NoError(t, err)
-	var res map[string]interface{}
+	var res requester.ContractAnswer
 	err = json.Unmarshal(body, &res)
 	require.NoError(t, err)
-	require.Contains(t, res["error"], "Incorrect signature")
+	require.Contains(t, res.Error.Message, "bad reference format")
```

Comment:
After changes this test doesn't check "IncorrectSign", but this case must be checked. Make changes in input data if needed and return this check.